### PR TITLE
1861 add endpoint for embeddable cards

### PR DESCRIPTION
**Pull request by solazio**
## Done
- _Add embeddable card endpoint and template._
- _Build embeddable cards page._
- _Fix a layout bug on the listing page._

## How to QA
- Run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/[charm_name]/publicise/cards
- Change the type of card by checking/un-checking things on the page. See it all works fine
- Go to http://localhost:8045/[charm_name]/listing and see the layout looks fine

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1861
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1862

## Screenshots
image

### CODE REVIEW

File: webapp/store/views.py [hunk 1]
```diff
@@ -486,3 +486,31 @@ def entity_badge(entity_name):
...
+    package = get_package(entity_name, channel_request, FIELDS)
+
+    package["default-release"]["channel"]["released-at"] = logic.convert_date(
+        package["default-release"]["channel"]["released-at"]
+    )
+
+    button_variants = ["black", "white"]
+    button = request.args.get("button")
+    if button and button not in button_variants:
+        button = "black"
```
Comment (jkfran) [inline webapp/store/views.py:503]: You can make this a one line:
```python
button = request.args.get("button", "black")
```

**Review by jkfran — APPROVED** [review]
LGTM :+1:

**Comment by solazio** [comment]
@spencerbygraves ehre is how the card looks like for varios states:

image

image

image

image

image

**Comment by spencerbygraves** [comment]
Thanks @solazio, a couple of things...

- The padding left and right should be 24px
- Please can we use the same grey pattern as in the header, the one in the card is darker

Thanks.

**Pull request merged**